stats/cloud/data.go

```diff
@@ -127,12 +127,14 @@ type SampleDataMap struct {
 // NewSampleFromTrail just creates a ready-to-send Sample instance
 // directly from a httpext.Trail.
 func NewSampleFromTrail(trail *httpext.Trail) *Sample {
+	tags := trail.GetTags()
+	tags.ReplaceURLWithName()
```

Comment:
This is not the right place to call `ReplaceURLWithName()` - it won't help with the aggregation of HTTP metrics, which was after all the main goal here. Look where `NewSampleFromTrail()` is called - only when [aggregation is disabled ](https://github.com/loadimpact/k6/blob/c2d73ba070b5223d60e784401e9bda8d634440fe/stats/cloud/collector.go#L276) or when non-aggregated outlier metrics are flushed.

Reply:
Fixed.